docs/UserGuide.md

```diff
@@ -27,12 +27,24 @@
 
 3. Copy the file to the folder you want to use as the _home folder_ for CFG.
 
-4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar contactsforgood.jar` command to run the application.<br>
-   
-5. A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.
+4. Open a command terminal, `cd` into the folder you put the jar file in.
+
+<box type="tip" seamless>
+
+* `cd` is the command to change directories.
+* e.g. `cd Documents/contactsforgood`
+
+</box> 
+
+5. Use the `java -jar contactsforgood.jar` command to run the application.
```

Comment:
I have noted that we do have
`1. Ensure you have Java 17 or above installed in your Computer.`
in the guide, but should we also tell the user to use `java -version` to check that they are running Java 17, much like how we conduct our smoke tests? We can tell them to refer back to `1.` if the version shown is lower than 17.

Or would that be too technical for an average user? Lmk your thoughts.

Reply:
Sounds good